src/interfaces/ISubscriber.sol

```diff
@@ -7,23 +7,15 @@ import {BalanceDelta} from "@uniswap/v4-core/src/types/BalanceDelta.sol";
 /// @notice Interface that a Subscriber contract should implement to receive updates from the v4 position manager
 interface ISubscriber {
     /// @param tokenId the token ID of the position
-    /// @param config details about the position
     /// @param data additional data passed in by the caller
-    function notifySubscribe(uint256 tokenId, PositionConfig memory config, bytes memory data) external;
+    function notifySubscribe(uint256 tokenId, bytes memory data) external;
     /// @param tokenId the token ID of the position
-    /// @param config details about the position
     /// @param data additional data passed in by the caller
-    function notifyUnsubscribe(uint256 tokenId, PositionConfig memory config, bytes memory data) external;
+    function notifyUnsubscribe(uint256 tokenId, bytes memory data) external;
```

Comment:
i feel like the subscriber would still want this information? imo it should still be sent

Reply:
subscriber can just read it tho so if they dont need it its worse. i think we should just remove it and now its consistent with notifyTransfer

Reply:
also it means more standardized to ERC721s not just PositionManager which is I think the point of this whole change no?

Reply:
yeah thats all fair, agreed